person.rb

```diff
@@ -0,0 +1,25 @@
+# The Person class represents a person with an id, name, and age.
+# The id is a unique identifier for the person.
+# The name is the name of the person.
+# The age is the age of the person.
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age, name: 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] the `id` argument you passed is going to give an error because it is not passed from the student or the teacher class, I advise you to remove it from the arguments and for now and create the instance variable with a random number.

Reply:
Resolved as advised